docs/operate/reference/components/camera/webcam.md

```diff
@@ -270,8 +270,8 @@ For more information about the vision service, see [run inference](https://docs.
 {{% /expand%}}
 
 {{% expand "High CPU usage" %}}
-Each camera stream you add uses CPU on the device it is connected to and there is therefore a practical limit to the numbeof camera streams your device can simultaneously support.
-You can limit the CPU usage by reducing the image resolution.
+Camera streams use a significant amount of CPU resources. The more CPU resources a device has, the more camera streams you can run simultaneously.
```

Comment:
```suggestion
Camera streams use a significant amount of CPU resources.
The more CPU resources a device has, the more camera streams you can run simultaneously.
```
[nittiest of nits] We sadly don't have a lint check for this currently but we try to keep newlines between sentences for ease of future diff comparison.

Sorry for drive-by review--wanted to see what your first PR was!